modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -242,12 +257,34 @@ inline void preprocess(const cv::Mat& src,
             dst = pp;
         }
 
+        if (with_pad) {
+            const auto div = ti.pad->divisibility;
+            const int padded_h = std::ceil(new_h / div) * div;
+            const int padded_w = std::ceil(new_w / div) * div;
+            int type = -1;
+            if (!is_hwc && new_c >= 1) {
+                type = ddepth;
+            } else {
+                type = ddepth == CV_32F ? CV_32FC3 : CV_8UC3;
+            }
+            GAPI_Assert(type != -1);
+            cv::Mat pad_im(cv::Size(padded_w, (!is_hwc ? new_c : 1) * padded_h), type, 0.f);
+            pad_im(cv::Rect(0, 0, dst.cols, dst.rows)) += dst;
```

Comment:
Why is it better to put `src` in the corner than in the center?

Reply:
[Preproc. steps](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn#preprocessing-steps)